src/diagnostics/LogPointsDebugAdapter.ts

```diff
@@ -114,6 +114,7 @@ export class LogPointsDebugAdapter extends LoggingDebugSession {
                     if (result.isSuccessful()) {
                         response.body = result.json;
                     } else {
+                        logger.error(`Cannot set logpoint, the error is: ${result.error}`);
```

Comment:
Nit: do you think "Cannot set logpoint. ${result.error}" would be a bit cleaner?

Reply:
Done